client/js/screens/SignIn/SignIn.js

```diff
@@ -74,17 +94,17 @@ const SignIn = ({ navigation }) => {
                 style={styles.logInButton}
                 onPress={handleSubmit}
               >
-                <Text style={styles.LogInText}> Log In</Text>
+                <Text style={styles.LogInText}>Log In</Text>
               </TouchableOpacity>
             </View>
           )}
         />
       </KeyboardAvoidingView>
-      <View style={styles.content}>
-        <TouchableOpacity onPress={() => navigation.navigate("Layout")}>
-          <Text style={styles.forgotPassword}>Forgot Password?</Text>
+      <View style={styles.contentContainer}>
+        <TouchableOpacity onPress={() => navigation.navigate("ResetPassword")}>
+          <Text style={styles.bodyText2}>Forgot Password?</Text>
```

Comment:
Show Auth errors from GraphCool (ex `User does not exist or Password is wrong`) if it exists.